Source/ToneAnalyzer/ToneAnalyzer.swift

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright IBM Corporation 2015
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Foundation
+import Alamofire
+import Freddy
+
+/**
+ * The IBM Watson The Tone Analyzer service uses linguistic analysis to detect 
+ * emotional tones, social propensities, and writing styles in written communication. 
+ * Then it offers suggestions to help the writer improve their intended language tones.
+**/
+public class ToneAnalyzer {
+    
+    private let username: String
+    private let password: String
+    private let versionDate: String
+    
+    private let serviceURL = "https://gateway.watsonplatform.net/tone-analyzer-beta/api"
+    private let tokenURL = "https://gateway.watsonplatform.net/authorization/api/v1/token"
+    private let errorDomain = "com.watsonplatform.toneanalyzer"
+    
+    public init(username: String, password: String, versionDate: String) {
```

Comment:
init needs to be documented since this is customer facing.  Jazzy shows Undocumented Declaration